scripts/jit/log_extract.py

```diff
@@ -122,43 +140,59 @@ def run_nvfuser(ir, inputs) -> float:
         return run_test(ir, inputs)
 
 
-def test_nvfuser(graphs: List[str], baseline_fn, nvfuser_fn):
+def test_runners(graphs: List[str], runners: List[Tuple[str, Callable]]):
     for i, ir in enumerate(graphs):
         _, inputs = load_graph_and_inputs(ir)
-        try:
-            baseline = baseline_fn(ir, inputs)
-            nvfuser = nvfuser_fn(ir, inputs)
-            improvement = (baseline / nvfuser - 1) * 100
-            print(f"  Graph {i}; baseline: {baseline:.2f} ms; nvfuser: {nvfuser:.2f} ms; improvement: {improvement:.2f}%")
-        except RuntimeError:
-            print(f"  Graph {i} failed:", traceback.format_exc())
+        print(f"Running Graph {ir}")
```

Comment:
nit - I thought it was nice having just the index, because it gets cluttered when we print the ir along with the performance results (especially with the nvfuser/torchbench results where there's ~2500 fusion groups). Plus it might be easier to export to csv if we're only printing indices.

But I'm fine with leaving in the IR if you think it's important

Reply:
Sure will update